package.json

```diff
@@ -90,12 +90,12 @@
     "ember-data": "^4.11.3",
     "ember-load-initializers": "^2.1.2",
     "ember-page-title": "^7.0.0",
-    "ember-qunit": "^6.2.0",
+    "ember-qunit": "^7.0.0",
```

Comment:
I believe this bump requires us to drop support for Ember 3.28. I'm not opposed to that, necessarily, but I wanted to call out the breaking change here which will require further changes in this PR unless you want to hold off on this bump for now.

Reply:
> this bump requires us to drop support for Ember 3.28

@elwayman02 not necessarily. we should be able just to use ember-qunit@6 via emebr-try scenario for Ember 3.28 (I did this in couple other addons that still support v3 and it's pretty easy doable)